subsys/bluetooth/host/hci_core.h

```diff
@@ -132,6 +132,43 @@ struct bt_le_ext_adv {
 #endif /* defined(CONFIG_BT_EXT_ADV) */
 };
 
+
+enum {
+	/** Periodic Advertising Sync has been created in the host. */
+	BT_PER_ADV_SYNC_CREATED,
+
+	/** Periodic advertising is in sync and can be terminated */
+	BT_PER_ADV_SYNC_SYNCED,
+
+	/** Periodic advertising is attempting sync sync */
+	BT_PER_ADV_SYNC_SYNCING,
+
+	BT_PER_ADV_SYNC_NUM_FLAGS,
+};
+
+struct bt_le_per_adv_sync {
+	/** Params used to establish sync */
+	struct bt_le_per_adv_sync_param params;
```

Comment:
Do you really need to store the parameters?

Reply:
Only some of it really (addr and sid). I think I used the options at one point too, which is the reason why I kept all of it at one point. But when it's only using 8/16 octets stored, I might as well just store the addr and SID :)